ui/src/components/ApprovalsTracker/index.tsx

```diff
@@ -172,21 +211,39 @@ const ApprovalsTracker: React.FC<Props> = (props): React.ReactElement => {
                                     )}
                                 </td>
                                 {user?.id === props.publication.createdBy && (
-                                    <td className="whitespace-nowrap py-4 px-6  text-sm text-grey-900 duration-500 dark:text-white-50">
-                                        {!author.linkedUser && (
-                                            <Components.IconButton
-                                                className="p-2"
-                                                title="Edit"
-                                                icon={
-                                                    <FaIcons.FaEdit
-                                                        className="h-4 w-4 text-teal-600 transition-colors duration-500 dark:text-teal-400"
-                                                        aria-hidden="true"
-                                                    />
-                                                }
-                                                onClick={() => setSelectedAuthor(author)}
-                                            />
-                                        )}
-                                    </td>
+                                    <>
+                                        <td className="whitespace-nowrap py-4 px-6  text-sm text-grey-900 duration-500 dark:text-white-50">
+                                            {!author.linkedUser && (
+                                                <Components.IconButton
+                                                    className="p-2"
+                                                    title="Edit"
+                                                    icon={
+                                                        <FaIcons.FaEdit
+                                                            className="h-4 w-4 text-teal-600 transition-colors duration-500 dark:text-teal-400"
+                                                            aria-hidden="true"
+                                                        />
+                                                    }
+                                                    onClick={() => setSelectedAuthor(author)}
+                                                />
+                                            )}
+                                        </td>
+                                        <td className="whitespace-nowrap py-4 px-6  text-sm text-grey-900 duration-500 dark:text-white-50">
+                                            {!author.confirmedCoAuthor && !author.reminderDate && (
+                                                <Components.IconButton
+                                                    className="p-2"
+                                                    disabled={isSendingReminder}
+                                                    title="Resend Email"
```

Comment:
might be better to label 'resend Email for {author email}' ?

Reply:
The ticket says 'Resend Email'. The email is already displayed on the same row

Reply:
We have an established , similar, pattern that does output the email in titles in the main co author edit page https://github.com/JiscSD/octopus/blob/main/ui/src/components/Publication/Creation/CoAuthorEntry/index.tsx

it is important for accessibility reasons, for both screen reader users as they may tab directly to the button and not even know the email address, and for partially sighted users who may not be able to see the email address in their field of view but can hover over the icon and see it is the right one.

Reply:
we will revisit this, OCT-570, and make sure accessibility is considered when refining